main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    dog
+//  action 1:    the dog barks
+dog.bark();
+//  action 2:    the dog whimpers
+dog.whimper();
+//  action 3:    the dog  runs
+dog.run();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    engine
+//  action 1:    engine accelerates
+engine.accelerate();
+//  action 2:    engine decelerates
+engine.decelerate();
+//  action 3:    engine neutral
+engine.neutral();

```

Comment:
`neutral` is not an action.  You need to include a verb in the action phrase.
- [ ] revise

Reply:
Replaced with verb.